src/constant.js

```diff
@@ -0,0 +1,24 @@
+import gdscLogo from "./assets/images/logo_GDSC.png";
+import youtubeLogo from "./assets/images/youtube_logo.jpeg";
+import csiLogo from "./assets/images/CSI_LOGO.png";
+
+export const resources = [
+  {
+    name: "GDSC Portal",
+    link: "https://gdsc.community.dev/conestoga-college-waterloo-campus/",
+    description: "You can view and RSVP to all of our upcoming events",
+    logo: gdscLogo,
+  },
+  {
+    name: "CSI club",
+    link: "https://conestogastudents.com/2021/09/01/coding-club/",
+    description: "You can register for the official GDSC Conestoga CSI club",
+    logo: csiLogo,
+  },
+  {
+    name: "YouTube",
+    link: "https://www.youtube.com/channel/UCBbpXl_fLG6dbu25nu2nRNA",
+    description: "You can watch all of our event on our youtube channel",
+    logo: youtubeLogo,
+  },
```

Comment:
The [doc](https://docs.google.com/document/d/1fkaTzob9xHglfr85g30PdW8Nv7wcwz9Doi1lZC_0rbE/edit?usp=sharing) we are aiming to replace has 7 (6 without the calendar) links with useful resources. Let's include all of them here. Let me know if you need any help with icons/text content.